portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -84,6 +88,64 @@ const addUkefFacilityIdToFacilities = async (applicationId) => {
   return facilities;
 };
 
+// retrieves user information from database
+const getUserInfo = async (userId) => {
+  const userCollectionName = 'users';
+
+  const userCollection = await db.getCollection(userCollectionName);
+  const {
+    firstname: firstName = '',
+    surname = '',
+  } = userId
+    ? await userCollection.findOne({ _id: new ObjectID(String(userId)) })
+    : {};
+
+  // creates user object which can be used
+  const user = {
+    firstname: firstName,
+    surname,
+    _id: userId,
+  };
+
+  return user;
+};
+
+// generates labels for portalActivities array based on type of submission
+const submissionTypeToConstant = (submissionType) => {
+  let submissionConstant;
+
+  switch (submissionType) {
+    case 'Automatic Inclusion Notice':
```

Comment:
We should have or add constants for the submission types

Reply:
done